src/neurostatslib/glm.py

```diff
@@ -1,13 +1,17 @@
+"""GLM core module
```

Comment:
are the only differences here the result of running the linters?

Reply:
yes